core/lib/server/render/pub/renderMarkdownWithPub.ts

```diff
@@ -37,12 +38,19 @@ const visitValueDirective = (node: NodeMdast & Directive, context: utils.RenderW
 	const field = expect(attrs.field, "Missing field attribute in value directive");
 
 	let value: unknown;
+	let pub: utils.RenderWithPubPub;
 
 	if (attrs?.rel === "parent") {
 		const parentPub = expect(context.parentPub, "Missing parent pub");
-		value = parentPub.values[field];
+		pub = parentPub;
 	} else {
-		value = context.pub.values[field];
+		pub = context.pub;
+	}
+
+	if (field === "title") {
+		value = getPubTitle(pub);
+	} else {
+		value = pub.values[field];
```

Comment:
I am open to that but hesitant to spend more time on it because I think the syntax for markdown directives is really just too complicated for most users. And I think having an example of this one at the top of every default form is better documentation than any of the other directives have!